Cab share/Manager/WeatherManager.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  WeatherManager.swift
+//  Cab share
+//
+//  Created by Marco Dell'Isola on 19/01/23.
+//
+
+import Foundation
+import CoreLocation
+
+class WeatherManager {
+    // MARK: - Request weather to Open Weather
+    func getCurrentWeather(latitude: CLLocationDegrees, longitude: CLLocationDegrees) async throws -> ResponseBody {
+        guard let url = URL(string:"http://api.openweathermap.org/data/2.5/weather?lat=\(latitude)&lon=\(longitude)&appid=760cb0970ae941895591c62f42d4e50e") else {
```

Comment:
@marcodellisola Why don't you use URLComonents to give queries to URL? Latitude and longtitude

Reply:
i didn't catch what you mean

Reply:
Did you do this with ChatGPT?:)

Reply:
It's better to use not URL but URLComponents here. As they taught us when you weren't at Academy

Reply:
nono, i did it chacking online resources, maybe chatgpt should did it well ahahah, anyway we should change it ?